src/client.ts

```diff
@@ -318,6 +318,7 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
       withCredentials: false, // making sure cookies are not sent
       warmUp: false,
       recoverStateOnReconnect: true,
+      disableCache: false,
```

Comment:
Can we add a check inside the client to only enable this when `this._isUsingServerAuth`? We won't test this on the client-side at all, so this might still be buggy there. I can see integrators enabling this trying to solve a different problem altogether causing weird to debug issues for us.

Reply:
I meant enable this if `disableCache  && this._isUsingServerAuth`

Reply:
Will update the jsdocs as well

Reply:
Nitpick: Instead of `disableCache,` can we use `enableCache`? It would save us from using double negations like `!disableCache`.

`if (enableCache)` reads better than `if (!disableCache)`

Reply:
It's already changed to that, changed it so that it includes the server side checks too

Reply:
Scratch that, hadn't pushed - NOW it's there